.circleci/config.yml

```diff
@@ -1,28 +1,61 @@
-version: 2
+version: 2.1
 jobs:
-  build:
+  validate-application-templates:
+    docker:
+      - image: circleci/buildpack-deps
+    steps:
+      - checkout
+      - run:
+          type: shell
+          name: Download openshift-validator-tool
+          command: |
+            wget https://github.com/jboss-container-images/jboss-kie-modules/raw/master/tools/openshift-template-validator/openshift-template-validator-linux-amd64
+      - run:
+          name: Add execution permission to openshift-validator-tool
+          command: |
+            chmod +x openshift-template-validator-linux-amd64
+      - run:
+          name: Print the openshift-validator-tool version
+          command: |
+            ./openshift-template-validator-linux-amd64 --version
+      - run:
+          name: Verify the templates
+          command: |
+            ./openshift-template-validator-linux-amd64 validate -d templates/
+      - run:
+          name: Verify the RHDM imagestream
+          command: |
+            ./openshift-template-validator-linux-amd64 validate --file rhdm78-image-streams.yaml
+
+  validate-cekit-image-yaml-files:
     docker:
-    - image: circleci/buildpack-deps
+      - image: circleci/buildpack-deps
```

Comment:
2 spaces of extra indentation were added here... probably accidently? The related PR in rhpam-7-openshift-image has the dash ("-") directly under the "d" in "docker", as it used to be here.

Reply:
updated.